examples/demo_workflow/app.py

```diff
@@ -0,0 +1,26 @@
+from dapr.ext.workflow.workflow_runtime_options import WorkflowRuntimeOptions
+from dapr.ext.workflow.workflow_client import WorkflowClient
+from dapr.ext.workflow.dapr_workflow_context import DaprWorkflowContext
+from dapr.ext.workflow.workflow_activity_context import WorkflowActivityContext
+
+def hello_world_wf(ctx: DaprWorkflowContext, input):
+    print(f'{input}')
+    yield ctx.call_activity(hello_act, input='  Boil one cup of water. ')
+    yield ctx.call_activity(hello_act, input='  Add tea leaves to the cup. ')
+    yield ctx.call_activity(hello_act, input='  Add boiled water to the cup. ')
+    yield ctx.call_activity(hello_act, input='  Add sugar and milk to the cup. ')
+
+def hello_act(ctx: WorkflowActivityContext, input):
+    print(f'{input}!', flush=True)
+
+daprOptions = WorkflowRuntimeOptions()
```

Comment:
The use of `Options` as the suffix for this class is confusing. Options are usually just a bag of properties. Instead, I think just `WorkflowRuntime` or `WorkflowApp` as the class name is good.

Reply:
Agreed, renamed to `WorkflowRuntime`.